jerry-core/ecma/operations/ecma-arraybuffer-object.c

```diff
@@ -60,6 +61,38 @@ ecma_arraybuffer_new_object (ecma_length_t length) /**< length of the arraybuffe
   return object_p;
 } /* ecma_arraybuffer_new_object */
 
+/**
+ * Helper function: create arraybuffer object with external buffer backing.
+ *
+ * The struct of external arraybuffer object:
+ *   ecma_object_t
+ *   extend_part
+ *   arraybuffer external info part
+ *
+ * @return ecma_object_t *
```

Comment:
`@return Pointer to the created ArrayBuffer object if success, NULL otherwise.`

Reply:
NULL can't be returned here